src/examples.md

```diff
@@ -101,6 +102,7 @@ or post [questions](https://github.com/mfem/mfem/issues/new?labels=question) or
       <option id="petsc">PETSc solvers</option>
       <option id="slepc">SLEPc eigensolvers</option>
       <option id="hiop">HiOp solvers</option>
+      <option id="pre">Preprocessing</option>
```

Comment:
Not sure this can be listed as a "solver" category

Reply:
I added this because example 30 currently shows up under every solver category, which is confusing.

An alternative could be that we find a way to make "none" the default option

Reply:
@tzanio do you have a preference on how to handle this?

Reply:
Maybe having a "none" option?